Integrations/integration-VirusTotal-Private_API.yml

```diff
@@ -1246,9 +1291,14 @@ script:
     arguments:
     - name: resource
       required: true
-      description: A URL for which you want to retrieve the most recent report. You
-        may also specify a scan_id (sha256-timestamp as returned by the URL submission
-        API) to access a specific report
+      description: A URL (single or multiple separated by a comma) for which you want
+        to retrieve the most recent report. You may also specify a scan_id (sha256-timestamp
+        as returned by the URL submission API) to access a specific report
+      isArray: true
+    - name: retries
+      description: Number of times the command will try to get url report if the report
+        was not ready on the first try.
```

Comment:
I think the last you should remove the dot in the end. and please review this description with @kirbles19

Reply:
:+1: